### ISSUE TITLE
Left can be float for offset() method

### ISSUE THREAD

**Issue by hardhub** [bridgedotnet/Frameworks#70]
Left can be float.... but Point has int.

```C#
namespace Bridge.jQuery2
{
  [External]
  [Name("Object")]
  [Constructor("{}")]
  public class Point
  {
    public readonly int Top;
    public readonly int Left;

    private Point()
    {
    }
  }
}
```
P.S. More info about offset: https://stackoverflow.com/questions/4626479/non-integer-offset-positions-in-jquery

**Comment by geoffreymcgill** [comment]
I agree this type should likely be modified to a `double`. This would trigger a breaking change, so it's going to be hard for us to justify revising at this point, but if there's any additional feedback from the community or other community maintainers, then we could make the change.

Our main development priority for supporting third-party frameworks is to improve the quality of [Retyped](https://retyped.com). Building Retyped is a difficult task and we certainly haven't reached the ideal state just yet, but we're working hard to get there. Any defects you can [report](https://github.com/Retyped/Retyped/issues) will be very helpful.

**Comment by hardhub** [comment]
> Our main development priority for supporting third-party frameworks is to improve the quality of Retyped. 

I see but for me it is possible to switch to Retyped only if a) it will give me any benefits b) it will be smooth migration.  

For now I do not have both a) code become more dirty b) I cannot even repeat the functionality already made with old packages.  

So I decided to clone old packages and improve them instead of switching to Retyped.  
For now it is just private repository. But in the future we can go with community (as you suggested above) and I will participate.  
It is my first project based on Bridge so first I will check how it is going.  

So to summarize all I think that Retyped is not good base for production use on current date.  

P.S. It will not be big breaking change. In worst case it will need just some cast. And in most cases it will not affect any code. And of course it is much more easier than migrate to Retyped.
